tools/windows/DatadogAgentInstaller/WixSetup/Datadog Agent/AgentInstaller.cs

```diff
@@ -145,6 +145,10 @@ public Project Configure()
                 {
                     AttributesDefinition = "Secure=yes"
                 },
+                new Property("DD_OTEL_OCI_INSTALL")
```

Comment:
option should be `DD_OTELCOLLECTOR_ENABLED` to match the other install scripts [[linux]](https://github.com/DataDog/agent-linux-install-script/blob/12a9032e5c8077b2ac918c5de5dbc732a6e99aac/install_script.sh.template#L1764) and [[fleet]](https://github.com/DataDog/datadog-agent/blob/072faaf6646e9ba41791eebacacfd753a1871f1a/pkg/fleet/installer/setup/defaultscript/default_script.go#L189-L195)